README.md

```diff
@@ -65,7 +65,7 @@ state files).
 [instance profiles]: http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html
 [aws-account-wide-terraform]: https://github.gds/government-paas/aws-account-wide-terraform
 
-* Declare you environment name using the variable DEPLOY_ENV. It must be 18 characters maximum and contain only alphanumeric characters and hyphens.
+* Declare your environment name using the variable DEPLOY_ENV. It must be 15 characters maximum and contain only alphanumeric characters and hyphens.
```

Comment:
Having looked at it again, I've realised that the README also says that you can only use "alphanumeric characters and hyphens". So we can't remove it as-is.

If you could extend the `Makefile` without too much difficulty to also check those, which would be beneficial, then I'd like the README entry removed. If it turns out to be too difficult though then don't worry and retain the README entry.